code/xcvm/lib/core/src/gateway/config.rs

```diff
@@ -155,7 +155,7 @@ pub struct InstantiateMsg(pub HereItem);
 #[cfg_attr(feature = "std", derive(schemars::JsonSchema))]
 pub struct HereItem {
 	/// Network ID of this network
-	pub id: NetworkId,
+	pub here_id: NetworkId,
```

Comment:
I’d probably go with `network_id` here as well.  The field describes network id of the ‘here’.

Reply:
will look at next comfig prs